### DISCUSSION TITLE
Welcome to Mesa Discussions!

### DISCUSSION THREAD

**Discussion by Corvince** [mesa/mesa#994] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tpike3** [comment]
Hi my name is Tom. My background is in computational social science, where I learned Agent Based Models at George Mason University.

My focus is trying to make the ABM ecosystem as robust and prevalent as computational ecosystems where users can rapidly build models so we can all gain deeper insight and understanding. (I am almost have some prototypes ready and will post once they are) 

For Mesa, I have helped with the tutorials and BatchRunnerMP. I am also  starting to look at the continuous space module to see if there is some optimizations to make Mesa faster.
